test/uk/gov/hmrc/transitmovements/controllers/ArrivalsControllerSpec.scala

```diff
@@ -323,4 +326,37 @@ class ArrivalsControllerSpec
     }
   }
 
+  "getArrivalsForEori" - {
+    val request = FakeRequest("GET", routes.ArrivalsController.getArrivalsForEori(eoriNumber).url)
+
+    "must return OK if arrivals were found" in {
+      val response = MovementWithoutMessages.fromMovement(movement)
+
+      when(mockRepository.getMovements(EORINumber(any()), any()))
```

Comment:
We should state the MovementType type used here, so for the getMovements calls, rather than any(), can we use eqTo(MovementType.Arrivals)?